document_page_procedure/README.rst

```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Procedures Template
+===================
+
+This module provides a Procedure Template.
+
+Installation
+============
+
+No installation steps required.
+
+Configuration
+=============
+
+No configuration steps required.
```

Comment:
To configure your template of procedures, you need to:

* go to Management Systems > Configuration > Categories
* select the Procedure category
* customize the template